site/docs/downloads/index.md

```diff
@@ -77,9 +77,9 @@ java -jar https://github.com/projectnessie/nessie/releases/download/nessie-{{ ve
 Requires Java 17 or newer.
 
 ```bash
-curl -o nessie-quarkus-cli-{{ versions.nessie }}-runner.jar \
-  https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
-java -jar nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
+curl -o nessie-admin-cli-{{ versions.nessie }}-runner.jar \
```

Comment:
This change needs to go into a separate PR (similar to #8468)

Reply:
#8484